src/Goerli-DssSpell.sol

```diff
@@ -46,19 +134,431 @@ contract DssSpellAction is DssAction {
+    address internal constant RWA011_A_INPUT_CONDUIT  = 0xcBd44c9Ec0D2b9c466887e700eD88D302281E098; // Tinlake Manager
+    address internal constant RWA011_A_OUTPUT_CONDUIT = 0xcBd44c9Ec0D2b9c466887e700eD88D302281E098; // Tinlake Manager
+
+    address internal constant MCD_JOIN_RWA012_A       = 0x0D9a5a31f16164e256E4f8b616c9C57F9d5C12d7;
+    address internal constant RWA012                  = 0x2E4378eF2A6822cfB0d154BA497B351e31C3B89b;
+    address internal constant RWA012_A_URN            = 0xa35F51d91311F60C904a02E1b0493Fc256A3F6e3;
+    address internal constant RWA012_A_OPERATOR       = 0xaef64c80712d5959f240BE1339aa639CDFA858Ff; // Tinlake Manager
+    address internal constant RWA012_A_INPUT_CONDUIT  = 0xaef64c80712d5959f240BE1339aa639CDFA858Ff; // Tinlake Manager
+    address internal constant RWA012_A_OUTPUT_CONDUIT = 0xaef64c80712d5959f240BE1339aa639CDFA858Ff; // Tinlake Manager
+
+    address internal constant MCD_JOIN_RWA013_A       = 0xD67131c06e93eDF3839C3ec5Bd92FF5D93A1e3df;
+    address internal constant RWA013                  = 0xc5Ac8B809a8De11D94b7Aa63b28b8fbBDF86Ea86;
+    address internal constant RWA013_A_URN            = 0xdC47d203753D3B5fb4fcD5900EBd96b0eC6761B6;
+    address internal constant RWA013_A_OPERATOR       = 0xc5A1418aC32B5f978460f1211B76B5D44e69B530; // Tinlake Manager
+    address internal constant RWA013_A_INPUT_CONDUIT  = 0xc5A1418aC32B5f978460f1211B76B5D44e69B530; // Tinlake Manager
+    address internal constant RWA013_A_OUTPUT_CONDUIT = 0xc5A1418aC32B5f978460f1211B76B5D44e69B530; // Tinlake Manager
 
     function actions() public override {
-        // ------------------ Setup new Starknet Governance Relay -----------------
-        // Forum: https://forum.makerdao.com/t/starknet-changes-for-executive-spell-on-the-week-of-2022-11-29/18818
-        // Relay l2 part of the spell
-        // L2 Spell:
-        // https://goerli.voyager.online/contract/0x05a958b692c498791a215f0624a105393f5b33cfdc3fb51bdf17e629b766119a#code
-        // it is a fix of 2022-11-28 l2 spell:
-        // https://goerli.voyager.online/contract/0x04c93f9818a4f81f6f2c6f0f660cb4986b789b6b6fb1b274b879649deed74eb8#code
-        // that did not executed correctly due to the incorrect bridge_legacy address:
-        // https://goerli.voyager.online/tx/0x6057a9ac5d03e09da99c9f07f725af5170eec6d4364f3a7f8dbc03362706698
-
-        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay(L2_GOV_RELAY_SPELL);
+
+        // Delegate Compensation - November 2022
+        // https://forum.makerdao.com/t/recognized-delegate-compensation-november-2022/19012
+        // NOT ON GOERLI
+
+
+        // Tech-Ops MKR Transfer
+        // https://mips.makerdao.com/mips/details/MIP40c3SP54
+        // NOT ON GOERLI
+
+
+        // MOMC Parameter Changes
+        // https://vote.makerdao.com/polling/QmVXj9cW
+
+        // Increase WSTETH-A line from 150 million DAI to 500 million DAI
```

Comment:
Subjective, but I would add line breaks in between all these different configurations.

Reply:
(not mandatory at this stage I guess, seeing the collateral offboarding stuff also uses the same condesed style)